### DISCUSSION TITLE
Question about Beta/Unstable 0.9.9

### DISCUSSION THREAD

**Discussion by wizard311** [jon-harper/OmniBox#67] [Q&A] [answered]
I try to back up your 0.9.9 files as they change and I found that PSUs folder is now missing was there an issue with the Mean Well LRS-350 file?  Second question is there plans for a Mean Well RSP-500-24 file?  I have 2 of these headed my way to install on my 2 printers and really want to drop them in the Omnibox to get rid of the Mean Well LRS-350 I currently run.  I can not open step files but I have included the drawing file from Mean Well hoping it has the mounting points included to help with file creation.

Thanks for all you do

[RSP-500-3D.zip](https://github.com/jon-harper/OmniBox/files/10036932/RSP-500-3D.zip)

**Comment by jon-harper** [comment, marked as answer]
My bad! I use GitHub for software projects so I'm really used to it. I need to do a better job accounting for that.

### What's Happening

First, let me kinda explain what I *think* is going on. GitHub is centered around file versions, and has "branches". I [wrote up short thread](https://twitter.com/TheOmniBox/status/1584092748029648898) about how git works and used OmniBox as an example. This may help explain things.

The pic below shows the OmniBox branches:

![image](https://user-images.githubusercontent.com/11852387/202629892-46ddd344-ca49-43ae-8de0-ac41af04b6d9.png)

`0.9.9-unstable` has the stuff that's not fit-tested (like the RSP-500 file). Once it's tested, it goes into `0.9.9-beta`.

GitHub's interface makes it too easy to switch back to the main branch, though. You might be in `0.9.9-unstable`, click the wrong link, and be back in the `main` branch. Another user had a similar issue and accidentally printed the wrong base, so clarifying this is definitely something I need to address.

### Fixing It

There's two easy ways to fix this for you:

- Download a Zip copy of the 0.9.9-unstable 
    - Has a straight copy of the files
    - Probably the simplest solution for now
- Use GitHub desktop
    - It's kinda like OneDrive or DropBox but for GitHub.
    - You can just browse the files on your hard drive.
    - Lets you set the branch you're working with, too.

Either way, go to the `0.9.9-unstable` branch and hit the green "Code" button and switch to the "Local" tab, like below. From there you can download the Zip or open the branch in GitHub Desktop.

![image](https://user-images.githubusercontent.com/11852387/202630366-30f63314-1ffb-4599-b9c5-e233cfddbc5c.png)

Lastly, [here's a direct link to the RSP-500 folder in the unstable branch](https://github.com/jon-harper/OmniBox/tree/0.9.9-unstable/Trays/PSU/Mean%20Well%20RSP-500).
